### ISSUE TITLE
Proposal : create new post command

### ISSUE THREAD

**Issue by k94ll13nn3** [Code52/pretzel#283]
Following #282, I thought it may be a good idea to have a pretzel command that will produce a new empty post file with a title that the user will provide, a file name using the YEAR-MONTH-DATE-TITLE.md format and a basic Yaml font matter.

**Example**
```
pretzel.exe newpost "A new post in Pretzel"
```
will produce `2015-12-04-a-new-post-in-pretzel.md` with the following font matter :
```
---
layout: post
title: "A new post in Pretzel"
date: 2015-12-04 21:00:00 +0100
comments: true
---
```
Not sure on what font matter could be included by default.

What do you think about this ?

**Comment by laedit** [comment]
Good idea.
About the front matter, if the date is in the path it is not needed as `date` variable.

**Comment by laedit** [comment]
Fixed with #284.
